doc/BUILDING.rst

```diff
@@ -1,12 +1,13 @@
 .. _build-native-client:
 
-Building Coqui STT Binaries
-===========================
+Building Binaries
+=================
 
-This section describes how to rebuild binaries. We have already several prebuilt binaries for all the supported platform,
-it is highly advised to use them except if you know what you are doing.
+This section describes how to build 🐸STT binaries.
 
-If you'd like to build the 🐸STT binaries yourself, you'll need the following pre-requisites downloaded and installed:
+You should **always** use our pre-built binaries, which are available with every `release <https://github.com/coqui-ai/STT/releases>`_, unless you have a very good reason to do build them yourself.
```

Comment:
So it's not always then? This language sounds condescending to me. Can you rephrase it as advice for users benefit instead of an order?

Reply:
I think this message needs to be very clear, so that no one accidentally wastes a day of their time trying to build something they never needed to build in the first place. Condescending is bad, obviously, so I'll make it more a clear suggestion than an order.